modules/dist/pom.xml

```diff
@@ -158,62 +117,6 @@
         <jboss.home>${project.build.directory}/wildfly-${jboss.version}</jboss.home>
       </properties>
     </profile>
-    <!--
-    Name:  deploy
-    Descr: Deploy stack on WildFly instance
-    -->
-    <profile>
-      <id>deploy</id>
-      <activation>
-        <property>
-          <name>!nodeploy</name>
```

Comment:
README should be updated on this parameter

Reply:
also docs https://github.com/jimma/jbossws-cxf/blob/galleon/docbook/src/main/doc/adoc/content/chapter-8-Build_and_testsuite_framework.adoc?plain=1#L127

Reply:
Fixed. Thanks.